lockbox-ios/View/ItemListView.swift

```diff
@@ -321,6 +325,9 @@ extension ItemListView {
         let searchImageView = UIImageView(frame: CGRect(x: 0, y: 0, width: 40.0, height: cell.frame.height))
         searchImageView.contentMode = .center
         searchImageView.image = searchImage
+        if #available(iOS 11.0, *) {
+            searchImageView.adjustsImageSizeForAccessibilityContentSizeCategory = true
```

Comment:
I think this pushed the image over to the left and looks cut off on the edge (should have some padding and shift back right?)